source/api/discovery/0.5/index.md

```diff
@@ -0,0 +1,915 @@
+        <ol>
+          <li>Retrieve and validate the representation of the page, and add the matching activities to <code class="highlighter-rouge">allActivities</code></li>
+        </ol>
+      </li>
+    </ol>
+  </li>
+  <li>Sort <code class="highlighter-rouge">allActivities</code> by <code class="highlighter-rouge">activity.endTime</code>, descending, removing any duplicates as described above.</li>
+  <li>For each activity in <code class="highlighter-rouge">allActivities</code>, apply the page algorithm's handling of an activity.</li>
+</ol>
+
+## 4. Network Considerations
+
+### 4.1. Activities for Access-Restricted Content
+
+Activities _MAY_ be published about content that has access restrictions. Clients _MUST NOT_ assume that they will be able to access every resource that is the object of an Activity, and _MUST NOT_ assume that if they cannot access it, then it has been deleted and therefore remove it from their indexes.  For example, the content might be protected by an authentication system that is denying access, or there might simply be a temporary network outage preventing the content from being retrieved. An end user might be able to provide the right credentials to gain access.
+
+Content may also change state from being available to being protected by access restrictions, or become available having previously been protected. There are no new activity classes for these situations, and the publisher _SHOULD_ issue the regular `Update` or `Delete` activities, if that is the behavior desired for harvesters.
+
+### 4.2. Negotiable Resources
+
+Some HTTP(S) URIs are able to respond with different representations of the same content in response to requests with different headers, such as the same URI being able to return both version 2 and version 3 of the IIIF Presentation API based on the Accept header. This is known as "content negotiation", and such resources are known as "negotiable resources". The representations that can be negotiated for are known as "variants".
+
+Negotiable resources are not supported by the Discovery API, only variants.  This means that there would be one Activity entry for each of the representations that are available, and that each representation _MUST_ have its own URI, even if it can also be reached via the negotiable resource.  In the case of negotiating for different versions of the IIIF Presentation API, the `format` property can be used to include the full media type of the resource, where the version specific context document is given in the `profile` parameter. If there is additional descriptive resources available, each such resource would describe all of the variants, and thus the `seeAlso` property of each Activity would refer to the same descriptions, allowing the variants to be connected together.
+
+Two variants of the same negotiable resource can be represented as follows.
```

Comment:
The two examples given below have different `id` values, so I would presume that they would be two different resources, instead of "the same negotiable resource."